docusaurus/docs/troubleshooting.md

```diff
@@ -0,0 +1,77 @@
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_position: 6
+---
+
+## Troubleshooting
+
+Here are some things to try and check if you experience problems while working with the `create-plugin` or `sign-plugin` tools in Windows.
```

Comment:
The Jest error is nothing to do with Windows so I think we'd need to change this as our intro.
```suggestion
Here are some things to try and check if you experience problems while working with the `create-plugin` or `sign-plugin` tools.
```